tests/metrics/test_popularity.py

```diff
@@ -104,3 +104,21 @@ def test_when_new_item_in_reco(self, interactions: pd.DataFrame, k: int, expecte
 
         actual_mean = arp.calc(reco, interactions)
         assert actual_mean == expected.mean()
+
+    @pytest.mark.parametrize(
+        "k,normalized,expected",
```

Comment:
If you don't use different `normalized` values, there is no need to keep it here. Just write `normalize=True` directly for `AvgRecPopularity`